packages/docs/pages/modal.tsx

```diff
@@ -123,6 +123,38 @@ const ModalPage = () => {
       <Panel header="Props" headerId="props">
         <ModalPropTable renderPanel={false} />
       </Panel>
+
+      <Panel header="Do's and Don'ts" headerId="props">
+        <GuidelinesTable
+          recommended={[
+            <>
+              <Code primary>Modals</Code> should require that users take an action.
+            </>,
+            <>
+              <Code primary>Modals</Code> should close when users press the Cancel or Primary button, not when merchants
+              click or tap the area outside the <Code primary>Modal</Code>.
+            </>,
+            <>
+              It should be clear why the <Code primary>Modal</Code> was opened.
+            </>,
+            <>
+              <Code primary>Modals</Code> should block other content on a page.
+            </>,
+            <>
+              User should have the option to come back without any <Code primary>actions</Code>.{' '}
+            </>,
+            <>Header and content should be left aligned, footer right aligned.</>,
+            <>
+              <Code primary>Modals</Code> should be positioned in the middle of the screen.
+            </>,
+          ]}
+          discouraged={[
+            <>
+              <Code primary>Modals</Code> shouldn’t have more than two buttons (primary and secondary) at the bottom.
```

Comment:
Not sure if `buttons` should have this:

```
<NextLink href="/button">Buttons</NextLink>
```

Reply:
no, since it's referencing a general "button"

only if we are referencing the `Button` component should we link/make it a `<Code />` element.